CHANGELOG.md

```diff
@@ -7,6 +7,45 @@
 
 # Release notes
 
+## 1.0.0
+
+### Changes
+
+- [breaking] Drop support for Android 4.4 [#169](https://github.com/medic/cht-android/issues/169).
+- [breaking] Remove Simprints integration [#230](https://github.com/medic/cht-android/issues/230).
```

Comment:
I think it'd be good to make a new `### Breaking changes` section for these two - they're kind of important.